src/pages/User/components/UserHeader/utils/handleAvatarUpload.ts

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+import { User } from 'src/utils/apollo.types';
+import { LocalStorage } from 'quasar';
+
+export const handleAvatarUpload = async (file: File | null | undefined) => {
+  if (!file) return null;
+
+  if (file.type.includes('image')) {
+    const url = 'https://www.did-admin.tk/api/files?prefix=user-profile';
```

Comment:
By the moments please add this in the object "env" of the quasar.config.js file, which will be changed in another PR for a .env file

![imagen](https://user-images.githubusercontent.com/66505715/191497592-5c9b948e-15ff-46a9-9302-997964ab3c1b.png)

So if you add it some like this:

```js
env: {
   GRAPHQL_URL: ......,
   NEXT_URL: "https://www.did-admin.tk/api"
}
```
You could use it here some like this:

```js
const url = process.env.NEXT_URL + "/files?prefix=user-profile"
```